inabox/tests/setup_test_harness.go

```diff
@@ -362,3 +553,30 @@ func newTransactOptsFromPrivateKey(privateKeyHex string, chainID *big.Int) *bind
 
 	return opts
 }
+
+func setupPaymentVaultTransactor(
```

Comment:
this is also a very generic/useful component right, that we could reuse as part of a CLI tool? Why have this under inabox?

Reply:
The transactor itself is provided by the bindings. This method is just to instantiate the existing transactor, so that tests can access it through the `TestHarness`. 

We might want a CLI to update the `PaymentVault` at some point, but this method here isn't directly applicable to anything beyond the scope of the `TestHarness`